### ISSUE TITLE
Fixed plugin.class.php

### ISSUE THREAD

**Issue by orlovskyjavaprofi** [MacWarrior/clipbucket-v5#277]
I fixed thanks to chatgpt this class plugin.class.php, because some plugins didnt installed properly and there were missing checks!

**[https://pastebin.com/pTf3r1ZR](https://pastebin.com/pTf3r1ZR)**

**Comment by MacWarrior** [comment]
Hello @orlovskyjavaprofi,

Thanks for your participation.
Can you pease be more precise about what issue(s) you fixed ?
We won't merge anything without suffisent details as working without merge request make things pretty hard to follow & check.
Most available plugins are "historical" and not compatible with CB 5.5 as specified on plugin page ; we have no known issue when installing and activating compatible plugins.

Thanks by advance,
Best regards,
MacWarrior.

**Comment by MacWarrior** [comment]
Hello @orlovskyjavaprofi, 

Thanks again for uour partitipation.

Be carefull as althrough ChatGPT is an interesting tool, it won't replace real development skills : code provided couldn't work as it removed "install_" prefix from plugin install filename generation.
Things have been adapted and corrected on latest commit.

Thanks again, 
Best regards,
MacWarrior.
